dali/pipeline/data/copy_to_external.h

```diff
@@ -44,39 +44,76 @@ struct kind2backend<mm::memory_kind::managed> {
 
 template <typename DstBackend, typename SrcBackend>
 inline void CopyToExternalImpl(void* dst,
-                               const Buffer<SrcBackend> &src,
+                               const Tensor<SrcBackend> &src,
                                cudaStream_t stream, bool use_copy_kernel) {
   DeviceGuard d(src.device_id());
   const auto &type_info = src.type_info();
   type_info.template Copy<DstBackend, SrcBackend>(dst, src.raw_data(), src.size(), stream,
                                                   use_copy_kernel);
 }
 
+template <typename DstBackend, typename SrcBackend>
+inline void CopyToExternalImpl(void* dst,
+                               const TensorList<SrcBackend> &src,
+                               cudaStream_t stream, bool use_copy_kernel) {
+  DeviceGuard d(src.device_id());
+  const auto &type_info = src.type_info();
+
+  // TODO(klecki): Add a proper test for non-contiguous access when we can have non-contiguous
```

Comment:
Do think we will have such case?
I imagine that DALI still should return raw outputs as continuous tensors so we can wrap them into CuPy/NumPy or DLPack directly.

Reply:
I just wrote it for the sake of TODO for the next PR mostly, I guess we won't be returning non-contiguous stuff soon, but either way I will add an error here or test this code-path and error out somewhere else.